src/components/SlotCalendarEvent/SlotCalendarEvent.tsx

```diff
@@ -19,16 +20,29 @@ export const SlotCalendarEvent: ComponentType<SlotCalendarEventProps> = ({ event
     const popupRef = useRef<HTMLDivElement>(null);
 
     const content = (
-        <div>
-            <Text size="xs" className={s.CreatorName}>
-                {creator?.name}
-            </Text>
-
-            <Text color="textSecondary" size="xs">
-                {title}
-            </Text>
+        <Text size="xs">
+            <Text>{creator?.name}</Text>
```

Comment:
Really needs text component inside the text component? 🙃

Reply:
If I use <p> I would have to reset default paragraph margins